src/ray/core_worker/experimental_mutable_object_provider.h

```diff
@@ -131,6 +131,22 @@ class MutableObjectProvider {
   /// \param[in] object_id The ID of the object.
   Status SetError(const ObjectID &object_id);
 
+  /// Checks if the channel is closed.
+  ///
+  /// \param[in] object_id The ID of the object.
+  /// \return Status indicating whether the object (if a channel for it exists) has its
+  ///         error bit set (i.e., the channel is closed).
+  Status IsChannelClosed(const ObjectID &object_id);
```

Comment:
`IsChannelClosed()` does not quite match the return type (Status), can we change it to `CheckChannelStatus()` across the board? With it, you may not need `IsExperimentalChannel()`.

Reply:
Done. I removed `IsExperimentalChannel()` from `MutableObjectProvider`.